src/server/messages.ts

```diff
@@ -48,40 +49,40 @@ export function validate(filename: string, project: string, text: string): Serve
     }
     const req = request(options);
     req.on('response', (res) => {
-        console.log("DONE");
+        log('info', "DONE");
 
         // DEBUG
-        console.log(`STATUS: ${res.statusCode}`);
-        console.log(`HEADERS: ${JSON.stringify(res.headers)}`);
+        log('debug', `STATUS: ${res.statusCode}`);
+        log('debug', `HEADERS: ${JSON.stringify(res.headers)}`);
         res.setEncoding('utf8');
         if (res.statusCode !== 200) {
-            console.error(`Request failed! Status: ${res.statusCode}`);
+            log('error', `Request failed! Status: ${res.statusCode}`);
             res.resume(); // Apparently needed to free up memory if we don't read the data?
         }
         // Parse the response to get the response ID
         res.on('data', (chunk) => {
             responseID = getResponseCode(chunk);
-            console.log(responseID);
+            log('debug', responseID);
             // Wait until the server has prepared the response
             if (commandSuccessful(responseID, options.host)) {
-                console.log(`Request ${responseID} is done.`);
+                log('info', `Request ${responseID} is done.`);
                 // Send Response message
                 // let ourResponse = createResponseMessage(responseID);
                 // TODO continue...
             } else {
-                console.log('Unsuccessful getting response.')
-            }
+                log('error', 'Unsuccessful getting response.')
+                }
```

Comment:
This change of indentation doesn't look right

Reply:
Fixed